clickhouse/client.go

```diff
@@ -0,0 +1,167 @@
+package clickhouse
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"time"
+
+	"github.com/ClickHouse/clickhouse-go/v2"
+	"github.com/ClickHouse/clickhouse-go/v2/lib/driver"
+	"github.com/chainbound/valtrack/log"
+	"github.com/rs/zerolog"
+)
+
+func ValidatorMetadataDDL(db string) string {
+	return fmt.Sprintf(`CREATE TABLE IF NOT EXISTS %s.validator_metadata (
+		peer_id String,
+		enr String,
+		multiaddr String,
+		ip String,
+		port UInt16,
+		last_seen UInt64,
+		last_epoch UInt64,
+		possible_validator Bool,
+		average_validator_count Int32,
+		num_observations UInt64
+	) ENGINE = MergeTree()
+PRIMARY KEY (peer_id, last_seen)`, db)
+}
+
+type ValidatorMetadataEvent struct {
```

Comment:
made a seperate type, for now. We can also merge it with `ParquetValidatorEvent` if we want them to have same fields.

Reply:
Let's have one struct that is the same for all events, and then use the tag annotations to make it compatible with multiple sinks (json, parquet, clickhouse)